src/modules/editor/reducer.ts

```diff
@@ -37,32 +50,50 @@ export const editorReducer = (state = INITIAL_STATE, action: EditorReducerAction
         mode
       }
     }
+
     case TOGGLE_PREVIEW: {
       const { enabled } = action.payload
       return {
         ...state,
         preview: enabled
       }
     }
+
     case TOGGLE_SIDEBAR: {
       const { enabled } = action.payload
       return {
         ...state,
         sidebar: enabled
       }
     }
+
     case SELECT_ENTITY: {
       return {
         ...state,
         selectedEntityId: action.payload.entityId
       }
     }
+
     case UNSELECT_ENTITY: {
       return {
         ...state,
         selectedEntityId: null
       }
     }
+
+    case EDITOR_READY: {
+      return {
+        ...state,
+        isReady: true
+      }
+    }
+
+    case CLOSE_EDITOR: {
+      return {
+        ...INITIAL_STATE
+      }
+    }
+
```

Comment:
I think this is fine but this would be the only file with newlines after the `}`. I'm annoying but I think it might be better to remove them for consistency sake